src/ai/backend/client/output/fields.py

```diff
@@ -61,6 +61,39 @@
     FieldSpec('mem_cur_bytes', 'Used Memory (MiB)', formatter=mibytes_output_formatter),
 ])
 
+domain_fields = FieldSet([
+    FieldSpec('name'),
+    FieldSpec('description'),
+    FieldSpec('is_active'),
+    FieldSpec('created_at'),
+    FieldSpec('total_resource_slots'),
+    FieldSpec('allowed_vfolder_hosts'),
+    FieldSpec('allowed_docker_registries'),
+    FieldSpec('integration_id'),
+])
+
+group_fields = FieldSet([
+    FieldSpec('id'),
+    FieldSpec('name'),
+    FieldSpec('description'),
+    FieldSpec('is_active'),
+    FieldSpec('created_at'),
+    FieldSpec('domain_name'),
+    FieldSpec('total_resource_slots'),
+    FieldSpec('allowed_vfolder_hosts'),
+    FieldSpec('integration_id'),
+])
+
+
+image_fields = FieldSet([
+    FieldSpec('name'),
+    FieldSpec('registry'),
+    FieldSpec('tag'),
+    FieldSpec('digest'),
+    FieldSpec('size_bytes'),
```

Comment:
Please add a new formatter based on `mibytes_output_formatter` by using `humanize.naturalsize(..., binary=True)` and apply it.